cypress/e2e/verifyLogin.cy.js

```diff
@@ -0,0 +1,13 @@
+/// <reference types= "cypress" />
+
+describe('Validate', function(){
+    
+    before(()=> {
+        cy.login();
+        
+        });
+    it('Validate that the new page is loaded', () => {
+        cy.get('.DweEFaF5owOe02').click();
+        cy.get('.lzFtVDCea8Z9jO').should('contain', 'staroz');
```

Comment:
This assertion will fail when other credentials are used. Try to do a more generic assertion.